src/Logging/MancyAppender.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using log4net.Appender;
+using log4net.Core;
+
+namespace Cimpress.Nancy.Logging
+{
+    public class MancyAppender : AppenderSkeleton

```

Comment:
You have several classes still named MancySomething. Do we want to rename those?